docs/REST API/rest-api.md

```diff
@@ -6327,20 +6328,25 @@ Body: <blob>
 
 _Available in Fleet Premium_
 
-List software that can or will be automatically installed during macOS setup. If `install_during_setup` is `true` it will be installed during setup.
+> `GET /api/v1/fleet/setup_experience/software` is deprecated as of Fleet 4.74.0. It is maintained for backwards compatibility. Please use `GET /api/v1/fleet/setup_experience/macos/software` to list macOS software for setup experience.
+
+List software that can or will be automatically installed during macOS or Linux setup. If `install_during_setup` is `true` it will be installed during setup.
```

Comment:
```suggestion
List software that can be automatically installed during setup. If `install_during_setup` is `true` it will be installed during setup.
```